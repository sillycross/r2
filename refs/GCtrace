<def f='luajit/lj_jit.h' l='251' ll='284'/>
<use f='luajit/lj_jit.h' l='251'/>
<size>120</size>
<doc f='luajit/lj_jit.h' l='250'>/* Trace object. */</doc>
<mbr r='GCtrace::nextgc' o='0' t='GCRef'/>
<mbr r='GCtrace::marked' o='64' t='uint8_t'/>
<mbr r='GCtrace::gct' o='72' t='uint8_t'/>
<mbr r='GCtrace::nsnap' o='80' t='uint16_t'/>
<mbr r='GCtrace::nins' o='96' t='IRRef'/>
<mbr r='GCtrace::unused_gc64' o='128' t='uint32_t'/>
<mbr r='GCtrace::gclist' o='192' t='GCRef'/>
<mbr r='GCtrace::ir' o='256' t='IRIns *'/>
<mbr r='GCtrace::nk' o='320' t='IRRef'/>
<mbr r='GCtrace::nsnapmap' o='352' t='uint32_t'/>
<mbr r='GCtrace::snap' o='384' t='SnapShot *'/>
<mbr r='GCtrace::snapmap' o='448' t='SnapEntry *'/>
<mbr r='GCtrace::startpt' o='512' t='GCRef'/>
<mbr r='GCtrace::startpc' o='576' t='MRef'/>
<mbr r='GCtrace::startins' o='640' t='BCIns'/>
<mbr r='GCtrace::szmcode' o='672' t='MSize'/>
<mbr r='GCtrace::mcode' o='704' t='MCode *'/>
<mbr r='GCtrace::mcloop' o='768' t='MSize'/>
<mbr r='GCtrace::nchild' o='800' t='uint16_t'/>
<mbr r='GCtrace::spadjust' o='816' t='uint16_t'/>
<mbr r='GCtrace::traceno' o='832' t='TraceNo1'/>
<mbr r='GCtrace::link' o='848' t='TraceNo1'/>
<mbr r='GCtrace::root' o='864' t='TraceNo1'/>
<mbr r='GCtrace::nextroot' o='880' t='TraceNo1'/>
<mbr r='GCtrace::nextside' o='896' t='TraceNo1'/>
<mbr r='GCtrace::sinktags' o='912' t='uint8_t'/>
<mbr r='GCtrace::topslot' o='920' t='uint8_t'/>
<mbr r='GCtrace::linktype' o='928' t='uint8_t'/>
<mbr r='GCtrace::unused1' o='936' t='uint8_t'/>
