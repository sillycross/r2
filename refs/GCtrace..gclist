<dec f='luajit/lj_jit.h' l='258' type='GCRef'/>
<offset>192</offset>
