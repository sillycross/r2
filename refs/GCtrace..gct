<dec f='luajit/lj_jit.h' l='252' macro='1' type='uint8_t'/>
<offset>72</offset>
<use f='luajit/lj_trace.c' l='132' u='w' c='lj_trace_alloc'/>
<use f='luajit/lj_trace.c' l='154' u='w' c='trace_save'/>
