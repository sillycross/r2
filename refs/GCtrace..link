<dec f='luajit/lj_jit.h' l='273' type='TraceNo1'/>
<use f='luajit/host/buildvm_arch.h' l='2355' u='a' c='build_subroutines'/>
<offset>848</offset>
<doc f='luajit/lj_jit.h' l='273'>/* Linked trace (or self for loops). */</doc>
<use f='luajit/lib_jit.c' l='308' u='r' c='lj_cf_jit_util_traceinfo'/>
<use f='luajit/lj_asm.c' l='2063' u='r' c='asm_tail_link'/>
<use f='luajit/lj_asm.c' l='2519' u='r' c='lj_asm_trace'/>
<use f='luajit/lj_gc.c' l='264' u='r' c='gc_traverse_trace'/>
<use f='luajit/lj_gc.c' l='264' u='r' c='gc_traverse_trace'/>
<use f='luajit/lj_record.c' l='295' u='w' c='lj_record_stop'/>
<use f='luajit/lj_record.c' l='301' u='w' c='lj_record_stop'/>
<use f='luajit/lj_trace.c' l='292' u='w' c='lj_trace_flushall'/>
<use f='luajit/lj_trace.c' l='522' u='w' c='trace_stop'/>
<use f='luajit/lj_trace.c' l='586' u='w' c='trace_abort'/>
<use f='luajit/lj_trace.c' l='594' u='w' c='trace_abort'/>
<use f='luajit/lj_trace.c' l='678' u='r' c='trace_state'/>
<use f='luajit/lj_trace.c' l='682' u='w' c='trace_state'/>
