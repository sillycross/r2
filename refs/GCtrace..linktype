<dec f='luajit/lj_jit.h' l='279' type='uint8_t'/>
<offset>928</offset>
<doc f='luajit/lj_jit.h' l='279'>/* Type of link. */</doc>
<use f='luajit/lib_jit.c' l='310' u='r' c='lj_cf_jit_util_traceinfo'/>
<use f='luajit/lib_jit.c' l='310' u='r' c='lj_cf_jit_util_traceinfo'/>
<use f='luajit/lj_record.c' l='294' u='w' c='lj_record_stop'/>
<use f='luajit/lj_record.c' l='1765' u='r' c='rec_func_jit'/>
<use f='luajit/lj_trace.c' l='595' u='w' c='trace_abort'/>
<use f='luajit/lj_trace.c' l='683' u='w' c='trace_state'/>
