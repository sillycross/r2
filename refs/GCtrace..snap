<dec f='luajit/lj_jit.h' l='262' type='SnapShot *'/>
<use f='luajit/lj_jit.h' l='294' u='r' c='snap_nextofs'/>
<offset>384</offset>
<doc f='luajit/lj_jit.h' l='262'>/* Snapshot array. */</doc>
<use f='luajit/lib_jit.c' l='367' u='r' c='lj_cf_jit_util_tracesnap'/>
<use f='luajit/lj_asm.c' l='966' u='r' c='asm_snap_alloc'/>
<use f='luajit/lj_asm.c' l='1031' u='r' c='asm_snap_prev'/>
<use f='luajit/lj_asm.c' l='1032' u='r' c='asm_snap_prev'/>
<use f='luajit/lj_asm.c' l='1042' u='r' c='asm_snap_fixup_mcofs'/>
<use f='luajit/lj_asm.c' l='2055' u='r' c='asm_tail_link'/>
<use f='luajit/lj_asm.c' l='2480' u='r' c='lj_asm_trace'/>
<use f='luajit/lj_opt_dce.c' l='25' u='r' c='dce_marksnap'/>
<use f='luajit/lj_opt_loop.c' l='147' u='r' c='loop_emit_phi'/>
<use f='luajit/lj_opt_loop.c' l='216' u='r' c='loop_subst_snap'/>
<use f='luajit/lj_opt_loop.c' l='296' u='r' c='loop_unroll'/>
<use f='luajit/lj_opt_loop.c' l='299' u='r' c='loop_unroll'/>
<use f='luajit/lj_opt_loop.c' l='308' u='r' c='loop_unroll'/>
<use f='luajit/lj_opt_loop.c' l='376' u='r' c='loop_unroll'/>
<use f='luajit/lj_opt_loop.c' l='378' u='r' c='loop_unroll'/>
<use f='luajit/lj_opt_loop.c' l='387' u='r' c='loop_undo'/>
<use f='luajit/lj_opt_loop.c' l='389' u='r' c='loop_undo'/>
<use f='luajit/lj_opt_sink.c' l='240' u='r' c='lj_opt_sink'/>
<use f='luajit/lj_record.c' l='1971' u='r' c='rec_comp_prep'/>
<use f='luajit/lj_record.c' l='1981' u='r' c='rec_comp_fixup'/>
<use f='luajit/lj_record.c' l='2019' u='r' c='lj_record_ins'/>
<use f='luajit/lj_record.c' l='2623' u='r' c='lj_record_setup'/>
<use f='luajit/lj_record.c' l='2637' u='r' c='lj_record_setup'/>
<use f='luajit/lj_snap.c' l='44' u='w' c='lj_snap_grow_buf_'/>
<use f='luajit/lj_snap.c' l='186' u='r' c='lj_snap_add'/>
<use f='luajit/lj_snap.c' l='192' u='r' c='lj_snap_add'/>
<use f='luajit/lj_snap.c' l='200' u='r' c='lj_snap_add'/>
<use f='luajit/lj_snap.c' l='326' u='r' c='lj_snap_shrink'/>
<use f='luajit/lj_snap.c' l='376' u='r' c='lj_snap_regspmap'/>
<use f='luajit/lj_snap.c' l='475' u='r' c='lj_snap_replay'/>
<use f='luajit/lj_snap.c' l='800' u='r' c='snap_unsink'/>
<use f='luajit/lj_snap.c' l='838' u='r' c='snap_unsink'/>
<use f='luajit/lj_snap.c' l='874' u='r' c='lj_snap_restore'/>
<use f='luajit/lj_trace.c' l='157' u='w' c='trace_save'/>
<use f='luajit/lj_trace.c' l='157' u='r' c='trace_save'/>
<use f='luajit/lj_trace.c' l='442' u='w' c='trace_start'/>
<use f='luajit/lj_trace.c' l='502' u='r' c='trace_stop'/>
<use f='luajit/lj_trace.c' l='509' u='r' c='trace_stop'/>
<use f='luajit/lj_trace.c' l='692' u='r' c='trace_state'/>
<use f='luajit/lj_trace.c' l='755' u='r' c='trace_hotside'/>
<use f='luajit/lj_trace.c' l='945' u='r' c='lj_trace_unwind'/>
